src/containers/About.jsx

```diff
@@ -10,89 +11,93 @@ const profileImage = name =>
 class About extends Component {
   render() {
     return (
-      <Container className="mt-5 mb-5">
```

Comment:
This looks like a lot of changes but it's just a <Fragment></Fragment> wrapping all the content.